2 - RTB/Documentazione esterna/Piano di Qualifica/Piano di Qualifica.typ

```diff
@@ -6,11 +6,6 @@
   authors: (
     "Todesco Mattia",
   ),
-  reviewers:(
```

Comment:
Perché hai rimosso i reviewers? Mi sembra che siano richiesti su tutte le copertine. Per i documenti più estesi credo che la cosa migliore sia fare la lista di tutti gli autori e tutti i reviewer (prendendo i dati dal log)

Reply:
I reviewers vengono inseriti in automatico prendendo da `roles.json`, in questo caso era stato esplicitato il reviewer perché altrimenti verificatore e redattore corrispondevano.
Ora però questo problema non c'è più visto che il verificatore è cambiato, però se mi dici posso fare un'ulteriore revisione di tutti i preamboli inserendo a mano la lista dei reviewer guardando le diverse PR. Facendo così, però, l'automatismo del loro inserimento non servirebbe più

Reply:
Hai ragione, scusami. Allora intanto teniamo così, e nel frattempo cerchiamo di capire se sia meglio:
1. Tenere solo il reviewer attuale
1. Fare una lista cumulativa dei reviewer
Questo vale anche per gli autori